controlpanel/frontend/views/app.py

```diff
@@ -66,7 +66,8 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         app = self.get_object()
 
-        context["app_url"] = cluster.App(app).url
+        # Temporarily hiding App URL until we fix the `KubernetesClient` bug causing the 401
+        context["app_url"] = None # cluster.App(app).url
```

Comment:
@andyhd not that familiar with Jinja but I'm assuming that setting `app_url` to `None` would mean the `if` in the template would evaluate `False` and don't show anything in the block, right? (sounds like a stupid question but worth double-checking)

```jinga
  {% if app_url %}
  <!--  DISPLAY URL HERE  -->
  {% endif %}
```

Reply:
Yes, the code in `{% ... %}` tags is evaluated as Python